ctf/tests/core_suite.lua

```diff
@@ -0,0 +1,91 @@
+local lunatest = require("lunatest")
```

Comment:
I think the test structure should be:

```
tests/
├── init.lua
├── core/
│      ├── core.lua
│      ├── teams.lua
│      └── diplo.lua
└──  <modname.lua>
```

Most mods will just have a single file, whereas bigger mods may have separate files